department-of-veterans-affairs/va.gov-cms
# VACMS-7070 out-of-box claro form style fixes

## Description
Relates to #7070

- fixed media cards for images/pdf
- removed divider from bottom of form above buttons
- made the delete button a scary red button
- updated list item bullets
- updated tags display for FAQ content type


- Did not adjust spacing issues for things like editorial workflow, hours of operation. Waiting until design tokens are finalized and we know what they should look like.
- Did not fix character count presentation for same reason, would like typographical design tokens.
- Did not adjust the select field length... the long one is because there is a very long name in the dropdown list ("Dr. Cameron McKinley Department of Veterans Affairs Veterans Center") and the field expands to that width.

## Testing done
- tested media with images, pdfs, multiple images (grid wraps after 3 now, all cards readable) on screen sizes 360px -> 1400px
- checked delete button is red & divider gone from node edit forms
- checked paragraph list bullet looks normal

## Screenshots
**before:**
<img width="914" alt="Screen Shot 2021-11-26 at 9 16 39 AM" src="https://user-images.githubusercontent.com/11279744/144137861-63b71d33-264a-4912-9b03-777d27d17264.png">

**after:**
bullet is normal size
<img width="775" alt="Screen Shot 2021-11-30 at 2 16 01 PM" src="https://user-images.githubusercontent.com/11279744/144137881-2d3ad6d6-c54e-42e4-bc24-b9c410fc292a.png">

**before:**
<img width="866" alt="Screen Shot 2021-11-26 at 10 08 27 AM" src="https://user-images.githubusercontent.com/11279744/144137930-63899a9f-c48d-4385-a04d-8222ba6f7f6d.png">

**after:** 
no line, button is red
<img width="843" alt="Screen Shot 2021-11-30 at 2 04 53 PM" src="https://user-images.githubusercontent.com/11279744/144137958-2162eb8b-9d2c-4c38-a3b7-a6e5d723c3c8.png">

**before:**
<img width="830" alt="Screen Shot 2021-11-26 at 10 15 35 AM" src="https://user-images.githubusercontent.com/11279744/144138181-429916c8-3e14-4c85-9d51-71c4d42efe62.png">
<img width="916" alt="Screen Shot 2021-11-26 at 10 29 02 AM" src="https://user-images.githubusercontent.com/11279744/144138184-db715c84-16ed-4fc8-b50b-fa85db55988b.png">

**after:**
card widened slightly, text given some space to breathe, pdf table fits in card 
<img width="801" alt="Screen Shot 2021-11-30 at 2 04 44 PM" src="https://user-images.githubusercontent.com/11279744/144138255-7afe965c-0c9e-4fee-9a74-879509252236.png">
<img width="823" alt="Screen Shot 2021-11-30 at 2 14 30 PM" src="https://user-images.githubusercontent.com/11279744/144138271-d62bf1c4-1938-4c74-b662-64bab04bb993.png">

**before:**
<img width="769" alt="Screen Shot 2021-11-26 at 12 52 30 PM" src="https://user-images.githubusercontent.com/11279744/144138540-3bb11369-5827-4bc4-b3a6-6e27bdebe097.png">

**after:**
<img width="834" alt="Screen Shot 2021-11-30 at 2 03 28 PM" src="https://user-images.githubusercontent.com/11279744/144138565-e22d8dab-4478-4b47-b36a-f682d745aa91.png">



## QA steps

As user _uid_ with _user_role_
- check Vet Center, VAMC System, FAQ, News Release types to see media updates & others look as expected

### Definition of Done


### Select Team for PR review


### CMS user-facing annoucement

Is an announcement needed to let editors know of this change?
